src/client.ts

```diff
@@ -2188,7 +2190,24 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
       ...options,
     });
   }
+  async blockUser(blockedUserID: string, user_id?: string) {
+    return await this.post<BlockUserResponse>(this.baseURL + '/users/block', {
+      blocked_user_id: blockedUserID,
+      ...(user_id ? { user_id } : {}),
+    });
+  }
 
+  async getBlockedUsers(user_id?: string) {
+    return await this.get<GetBlockedUsersResponse>(this.baseURL + '/users/block', {
```

Comment:
Can you change `GetBlockedUsersResponse` to `GetBlockedUsersAPIResponse`? Same for BlockUserResponse